src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  grid-template-columns: 100%;
+  width: $card-width;
+  background-color: $color-wight;
```

Comment:
There is a typo in the variable name, it should be $color-white instead of $color-wight.